### PR TITLE
/oauth2/keys Specify the service to obtain the public key

### PR DESCRIPTION
**Pull request by TakuyaMatsu**
# Description
#2638
`/oauth2/keys` query parameter to specify a service so that the public key of the specified service can be obtained.
e.g. `/oauth2/keys?rfc=true&service=zms`

# Contribution Checklist:
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have read the [contribution guidelines](https://github.com/AthenZ/athenz/blob/master/CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request.**

### CODE REVIEW

File: servers/zts/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <packaging>war</packaging>
 
   <properties>
-    <code.coverage.min>0.9954</code.coverage.min>
+    <code.coverage.min>0.9952</code.coverage.min>
```
Comment (havetisyan) [inline servers/zts/pom.xml:32]: we do not allow reducing the code coverage. please review the jacoco output and add test cases to have full 100% coverage for any new code.

File: servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java [hunk 2]
```diff
@@ -5020,13 +5020,18 @@ public ExternalCredentialsResponse postExternalCredentialsRequest(ResourceContex
...
-    public JWKList getJWKList(ResourceContext ctx, Boolean rfc) {
+    public JWKList getJWKList(ResourceContext ctx, Boolean rfc, String service) {
 
         final String caller = ctx.getApiName();
         final String principalDomain = logPrincipalAndGetDomain(ctx);
 
         validateOIDCRequest(ctx.request(), principalDomain, caller);
-        return dataStore.getZtsJWKList(rfc);
+        switch (service) {
+          case "zms":
```
Comment (havetisyan) [inline servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java:5030]: let's replace "zms" with ServerCommonConsts.ZMS_SERVICE

**Review by havetisyan — CHANGES_REQUESTED** [review]

File: servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java (hunk 2 above)
Comment (TakuyaMatsu) [reply servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java]: fixed

File: servers/zts/pom.xml (hunk 1 above)
Comment (TakuyaMatsu) [reply servers/zts/pom.xml]: fixed

**Review by havetisyan — APPROVED** [review]

**Pull request merged**